boards/trellis_m4/examples/neopixel_accel.rs

```diff
@@ -9,10 +9,10 @@ extern crate smart_leds;
 extern crate trellis_m4 as hal;
 extern crate ws2812_nop_samd51 as ws2812;
```

Comment:
I think you should get rid of these extern crates as part of 2018 edition upgrade.

Reply:
Absolutely... not sure why `cargo fix --edition-idioms` missed them

Reply:
I got rid of the trivially unnecessary ones in 5561a3e7c.

We still need `extern crate panic_halt` to define a `#[panic_handler]`:

```
   Compiling trellis_m4 v0.1.2 (/Users/bascule/src/atsamd/boards/trellis_m4)
error: `#[panic_handler]` function required, but not found
```

I can go ahead and get rid of the use of aliased crate names in the examples at least...

Reply:
Took another stab in 9ce402212 and force pushed. That should be all of them except for the `extern crate panic_halt`, which if I try to replace with just `use panic_halt` I get the following:

```
warning: unused import: `panic_halt`
 --> examples/neopixel_rainbow.rs:4:5
  |
4 | use panic_halt;
  |     ^^^^^^^^^^
  |
  = note: #[warn(unused_imports)] on by default
```

Reply:
I think you can #[allow(unused)] on there.

Reply:
Done in 1698bd484